GVRf/Framework/framework/src/main/java/org/gearvrf/scene_objects/GVRGearControllerSceneObject.java

```diff
@@ -29,18 +31,9 @@
  * Lastly, one can use {@link #enableSurfaceProjection()} to project the cursor onto the
  * object being picked. Read the method documentation for more information.
  */
-
-public class GVRGearControllerSceneObject extends GVRSceneObject {
-    private final String TAG = GVRGearControllerSceneObject.class.getSimpleName();
-    private float rayDepth = 1.0f;
-    private boolean projectToSurface = false;
-    private GVRBaseSensor disabledSensor;
-    private GVRSceneObject cursor = null;
-    private GVRCursorController gearCursorController = null;
-    private SensorListener sensorListener = new SensorListener();
+public class GVRGearControllerSceneObject extends GVRCursorControllerSceneObject
```

Comment:
The GVRGearControllerSceneObject is so thin now that in my opinion it should be rolled into the GVRCursorControllerSceneObject as something you could optionally enable.